cdap-ui/app/login/login.js

```diff
@@ -65,7 +65,7 @@ class Login extends Component {
         }
       })
       .then((res) => {
-        cookie.save('CDAP_Auth_Token', res.access_token);
+        cookie.save('CDAP_Auth_Token', res.access_token, { path: '/'});
```

Comment:
The path is mentioned here because the cookies are namespaced by domain name. Right now we serve different webapps for `/cdap`, `/hydrator/` & `/tracker/`. So if we want to share the cookie between them we need to add it to `/` path so that they are visible in all webapps.